bucket/flyctl.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "0.0.143",
+    "description": "Flyctl Deployment/Management Client",
+    "homepage": "https://github.com/superfly/flyctl",
+    "license": " Apache-2.0-License",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/superfly/flyctl/releases/download/v0.0.143/flyctl_0.0.143_Windows_x86_64.tar.gz",
+            "hash": "F3F5BEFBA3583796C0C268F45FBB77AEAB881EE00E860752B30266E5CE8AD782"
+        }
+    },
+    "bin": "flyctl.exe",
```

Comment:
> Note: Most installations of flyctl also alias flyctl to fly as a command name and this will become the default name in the future. During the transition, note that where you see flyctl as a command it can be replaced with fly.

```suggestion
    "bin": [
        "flyctl.exe",
        [
            "flyctl.exe",
            "fly"
        ]
    ],
```